src/transformState.js

```diff
@@ -6,6 +6,32 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const act of actions) {
+    const { type, extraData, keysToRemove } = act;
+
+    switch (type) {
+      case 'addProperties':
+        Object.assign(state, extraData);
+        break;
+
+      case 'removeProperties':
+        for (const i of keysToRemove) {
```

Comment:
Let's name variables appropriately
```suggestion
        for (const key of keysToRemove) {
```